### ISSUE TITLE
Error on https://gamejolt.net/

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#14170]
# Error Info
Error name: Error
Error message: panicked at /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/49d16f7/wgpu-core/src/resource.rs:207:13:
Map callback was leaked
Error stack:
```
339/__wbg_get_imports/imports.wbg.__wbg_new_ab87fd305ed9004b@moz-extension://862c35d8-4b18-4317-b90f-ae1aaf845783/dist/339.js:2624:21
core::ops::function::FnOnce::call_once{{vtable.shim}}::hb1c47d692dbcedf6@:wasm-function[9803]:0x9b0c0a
std::panicking::rust_panic_with_hook::hc20eadded6bfe687@:wasm-function[6278]:0x89aea5
core::panicking::panic_fmt::h7a368385936888dc@:wasm-function[9603]:0x9ab9e1
<wgpu_core::resource::BufferMapCallback as core::ops::drop::Drop>::drop::hc09fcb734b2606c9@:wasm-function[8946]:0x967683
<T as wgpu::context::DynContext>::surface_configure::h8e332589e067789b@:wasm-function[8114]:0x9307d5
wgpu::Surface::configure::h2f45366185fb928f@:wasm-function[4133]:0x78af8a
<ruffle_render_wgpu::target::SwapChainTarget as ruffle_render_wgpu::target::RenderTarget>::resize::haf93e9196a23cae1@:wasm-function[10166]:0x9c59c5
<ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::set_viewport_dimensions::h5e3a21b70875f5a8@:wasm-function[5107]:0x832131
ruffle_core::player::Player::set_viewport_dimensions::ha6af1a0ea287cac4@:wasm-function[1560]:0x4ba751
ruffle_web::Ruffle::tick::hf791ff8ec78789eb@:wasm-function[861]:0x2e37d8
ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::h70bac6e8d2ef615c@:wasm-function[11942]:0x9fedf8
wasm_bindgen::convert::closures::invoke1_mut::h0c5d8d34f590675d@:wasm-function[10506]:0x9cbfbb
__wbg_adapter_66@moz-extension://862c35d8-4b18-4317-b90f-ae1aaf845783/dist/339.js:294:10
real@moz-extension://862c35d8-4b18-4317-b90f-ae1aaf845783/dist/339.js:271:20

```

# Player Info
Allows script access: false
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (Intel, Intel(R) HD Graphics Direct3D11 vs_5_0 ps_5_0)"
Adapter Device Type: IntegratedGpu
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_COMPRESSION_BC | TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS | SHADER_UNUSED_VERTEX_OUTPUT)
Available features: Features(CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 16384, max_texture_dimension_2d: 16384, max_texture_dimension_3d: 2048, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 1000, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 960, height: 720, depth_or_array_layers: 1 }
Player type: Object
SWF URL: https://download.gamejolt.net/4b08a5b9f3b80652e6ea41bc4cac2539e0ac45427066f2e9c01f390a38483a29,1701066070,7/data/games/0/2/12752/files/5419be8e2eb1a/CantEscape.swf
Param allowFullScreen: true
Param allowFullScreenInteractive: true
Param wmode: direct
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined

# Page Info
Page URL: https://gamejolt.com/games/i-can-t-escape/12752
SWF URL: https://download.gamejolt.net/4b08a5b9f3b80652e6ea41bc4cac2539e0ac45427066f2e9c01f390a38483a29,1701066070,7/data/games/0/2/12752/files/5419be8e2eb1a/CantEscape.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:120.0) Gecko/20100101 Firefox/120.0
Platform: Win32
Has touch support: true

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-11-17
Channel: nightly
Built: 2023-11-17T00:14:46.670Z
Commit: a78a67ee24a4a172e6fae0ce03979e63864270c6
Is extension: true

# Metadata
width: 640
height: 480
frameRate: 24
numFrames: 2
swfVersion: 18
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 6952015

**Comment by n0samu** [comment]
The problem also happens in the latest version of Ruffle on desktop and web. In Firefox it happens while the game enters fullscreen, while in Chromium and on desktop it happens when it exits fullscreen. Related to #13543 and #14051.

**Comment by torokati44** [comment]
I was only able to reproduce in Chrome (not in Firefox or on Desktop), and it seems that #13732 will fix this.

**Comment by n0samu** [comment]
Fixed by #13732
